evm/exceptions.py

```diff
@@ -151,14 +151,6 @@ class OutOfBoundsRead(VMError):
     pass
 
 
-class UnannouncedStateAccess(VMError):
-    """
-    Error raised if a storage slot or account field is accessed that has not
-    been specified in the transaction's read or write list, respectively.
-    """
-    pass
-
-
 class GasPriceAlreadySet(Exception):
```

Comment:
Should this get remove since I believe it is `PAYGAS` specific.

Reply:
:+1: missed this (same for `NotTopLevelCall`)